app/frontend/src/filters/index.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue';
+import moment from 'moment';
```

Comment:
Just read an article about how this lib is getting deprecated (though will continue to be updated with timezone changes for the next few years), should keep in mind for all our apps for the future.
https://momentjs.com/docs/#/-project-status/

Reply:
uh oh...